### PR TITLE
Added Circle CI extension

### PR DESCRIPTION
**Pull request by qeude**
## Description

Add a Circle CI extensions. It lets you list your workflows and their associated jobs.

## Type of change

- New extension

## Screencast

Screen Recording 2021-12-04 at 18 27 50

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/circle-ci/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+
+
+function getWorkflowActions(workflow: Workflow) {
+  const workflowUrl = `https://app.circleci.com/pipelines/workflows/${workflow.id}`;
+  if (!workflow.repository.target_repository_url) {
+    return (
+      <ActionPanel>
+        <PushAction title="Workflow's jobs list" target={<JobList workflow={workflow} />} />
+        <OpenInBrowserAction title="Open workflow" url={workflowUrl} />
+        <CopyToClipboardAction title="Copy workflow URL" content={workflowUrl} />
```
Comment (pernielsentikaer) [inline extensions/circle-ci/src/index.tsx:215]:
```suggestion
        <PushAction title="Workflow Job List" target={<JobList workflow={workflow} />} />
        <OpenInBrowserAction title="Open Workflow" url={workflowUrl} />
        <CopyToClipboardAction title="Copy Workflow URL" content={workflowUrl} />
```
Also icon is missing in the PushAction

File: extensions/circle-ci/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+      </ActionPanel>
+    )
+  } else {
+    return (
+      <ActionPanel>
+        <PushAction title="Workflow's jobs list" target={<JobList workflow={workflow} />} />
+        <OpenInBrowserAction title="Open workflow" url={workflowUrl} />
+        <CopyToClipboardAction title="Copy workflow URL" content={workflowUrl} />
```
Comment (pernielsentikaer) [inline extensions/circle-ci/src/index.tsx:223]:
```suggestion
        <PushAction title="Workflow Job List" target={<JobList workflow={workflow} />} />
        <OpenInBrowserAction title="Open Workflow" url={workflowUrl} />
        <CopyToClipboardAction title="Copy Workflow URL" content={workflowUrl} />
```
Also icon is missing in the PushAction

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @qeude 👋 

Thanks for you contribution, very nice! 💯 

I have now tested you extensions and I have a few comments and suggestions:
- If credentials is wrong, I suggest you to stop the loading and show "No Result" instead
- Remove console.log's

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by qeude** [comment]
Hi @pernielsentikaer,
Thanks for your feedback, everything should be good now. 🙌🏼

**Review by pernielsentikaer — APPROVED** [review]
Hi @qeude 👋

Thanks for the update, much better 💪 

I'm going to accept this now, my colleague will do a final review before merging it 💯

**Pull request merged**